dspace-api/src/test/java/org/dspace/app/csv/CSVMetadataImportReferenceIT.java

```diff
@@ -19,54 +19,132 @@
 import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.CollectionService;
+import org.dspace.content.service.CommunityService;
+import org.dspace.content.service.EntityTypeService;
+import org.dspace.content.service.InstallItemService;
 import org.dspace.content.service.ItemService;
 import org.dspace.content.service.MetadataFieldService;
 import org.dspace.content.service.MetadataValueService;
 import org.dspace.content.service.RelationshipService;
+import org.dspace.content.service.RelationshipTypeService;
+import org.dspace.content.service.WorkspaceItemService;
+import org.dspace.scripts.DSpaceRunnable;
+import org.dspace.scripts.configuration.ScriptConfiguration;
+import org.dspace.scripts.factory.ScriptServiceFactory;
+import org.dspace.scripts.service.ScriptService;
+import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-import org.springframework.beans.factory.annotation.Autowired;
 
 /**
  * Created by: Andrew Wood
  * Date: 26 Jul 2019
  */
-public class CSVMetadataImportReferenceIT extends AbstractEntityIntegrationTest {
+public class CSVMetadataImportReferenceIT extends AbstractIntegrationTest {
 
     //Common collection to utilize for test
     private Collection col1;
 
-    @Autowired
-    private RelationshipService relationshipService;
+    private RelationshipService relationshipService = ContentServiceFactory.getInstance().getRelationshipService();
+    private ItemService itemService = ContentServiceFactory.getInstance().getItemService();
+    private CollectionService collectionService = ContentServiceFactory.getInstance().getCollectionService();
+    private CommunityService communityService = ContentServiceFactory.getInstance().getCommunityService();
+    private WorkspaceItemService workspaceItemService = ContentServiceFactory.getInstance().getWorkspaceItemService();
+    private InstallItemService installItemService = ContentServiceFactory.getInstance().getInstallItemService();
+    private EntityTypeService entityTypeService = ContentServiceFactory.getInstance().getEntityTypeService();
+    private RelationshipTypeService relationshipTypeService = ContentServiceFactory.getInstance()
+                                                                                   .getRelationshipTypeService();
 
-    @Autowired
-    private ItemService itemService;
+
+    Community parentCommunity;
 
     /**
      * Setup testing enviorment
      */
     @Before
-    public void setup() {
+    public void setup() throws SQLException, AuthorizeException {
+        context.turnOffAuthorisationSystem();
+        parentCommunity = communityService.create(null, context);
+        communityService.addMetadata(context, parentCommunity, "dc", "title", null, null, "Parent Community");
+
+        col1 = collectionService.create(context, parentCommunity);
+        collectionService.addMetadata(context, col1, "dc", "title", null, null, "Collection 1");
```

Comment:
We should be using `Builder` classes in this `setup()` method to create *all* new content.  That would mean that we can completely *remove* the `destroy()` class (as Builders cleanup after themselves). 

It looks like the old code in this `setup()` used Builders, but you've replaced/removed that code.  Please revert back to using `Builders` again to create the Community, Collection (see above) and all Entities (see below). 

**If you rebase on `main`, the Builder classes are now available in `dspace-api`.**